index.html

```diff
@@ -60,7 +57,173 @@ <h1 class="greet">
+            <img src="assets/Frame 2.png">
+          </div>
+          <div class="work-info">
+            A daily selection of private and personalised<br>
+            reads ; no accounts<br>
+            or sign-ups required.
+          </div>
+          <div class="langs">
+            <ul class="lang-list">
+              <li class="Html">html</li>
+              <li class="Css">css</li>
+              <li class="js">javascript</li>
+            </ul>
+          </div>
+          <div class="see-project">
+            <button class="button">See Project</button>
+          </div>
+      </div>
+      </section>
+      <div class="aboutme_container">
+        <p class="about_me" #id="about_me">About<br> Myself</p>
+        <p class="intro_2">
+          I'm a software developer ! <br />
+          I make websites , features and other projects<br />
+          have a look at my work.<br />
+          May be you might like it,<br />
+          if you do then please contact me on:
+          <br /><br />
+        </p>
+        <p class="social-text_2">Let's connect</p><br><br>
+        <ul class="social-icon_2">
+          
+          <li class="twitter-handle">
+            <a href="https://twitter.com/RishiMi31357764"
+              ><i class="fa fa-twitter"></i
+            ></a>
+          </li>
+          <li class="linkedin-handle">
+            <a href="https://www.linkedin.com/in/rishi-mishra-756718257/"
+              ><i class="fa fa-linkedin"></i
+            ></a>
+          </li>
+          <li><a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+          <li class="github-handle">
+            <a href="https://github.com/Rishi-Mishra0704"
+              ><i class="fa fa-github"></i
+            ></a>
+          </li>
+          <li><a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>    
+        </ul>
+        <div class="resume">
+          <button class="button" id="resume">Get my resume</button>
+        </div>
 
+        
+          <ul class="skills">
+              <ul class="languages">  
+                <li><p>Languages</p></li>
+                <li><img src="assets/Unionlangs.png"></li>
+              </ul>
+              <li class="js_2"> 
+                <img src="assets/jsLogo.png"><p>Javascript</p>
+              </li>
+              <li class="Html_2">
+                <img src="assets/htmlLogo.png"><p>Html</p>
+              </li>
+              <li class="Css_2">
+                <img src="assets/cssLogo.png"><p>Css</p>
+              </li>
+            </ul>
+            
+            <ul class="framework">
+            <li >
+              <p>FrameWork</p>
+            </li>
+            <li><img src="assets/Unionarrow.png"></li>
+            </ul>
+            <ul class="skill_txt">
+              <li>
+                <p>
+                  Skills
+                </p>
+              </li>
+              <li><img src="assets/Unionarrow.png"></li>
+          </ul>
```

Comment:
- [ ] Kindly fix the alignment of the `Languages`, `Frameworks`, and `Skills` to match the given [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=TwQwsTNCrgvEHLHB-0). All items should have same left margin/space.
![Screenshot from 2023-01-13 22-08-20](https://user-images.githubusercontent.com/60083437/212366619-112ac2cb-ec7f-484e-b5a0-0bb986258a0b.png) 

- [ ] Kindly fix the `fonts weight`, `fonts weight`, and `fonts color` to match the given design.